environment.yaml

```
@@ -2,30 +2,16 @@ name: openfe-notebooks
 channels:
   - conda-forge
 dependencies:
-  - MDAnalysis
-  - click
   - coverage
   - ipywidgets
-  - lomap2
   - lxml
-  - mdtraj
   - nbval
-  - networkx
   - nglview
   - notebook
-  - openff-forcefields
-  - openmm
-  - openmmtools>=0.23.0
   - pip
-  - plugcli
-  - pymbar
   - pytest
   - pytest-cov
   - pytest-xdist
-  - pydantic
   - python
-  - rdkit
-  - typing_extensions
   - gufe>=1.7.0
   - openfe>=1.7.0
```

```suggestion
  - openfe>=1.8.0
```

Should we use the new release? Also I wonder if we should be using `==` for `gufe` and `openfe` since these notebooks are really tied to the API of those packages (or demo new features that don't exist in older code versions).